docs/topics/api/index.rst

```diff
@@ -2,8 +2,26 @@
 External API
 ============
 
-This is documentation for the API available at `/api/v3/`.
+This shows you how to use the `addons.mozilla.org <https://addons.mozilla.org/en-US/firefox/>`_
```

Comment:
and I really hate having a locale in the URL but someone convinced themselves long ago that we need it for caching